warp10/src/main/java/io/warp10/script/filter/FilterByLabels.java

```diff
@@ -136,12 +136,12 @@ public FilterByLabels(String name, Map<String,String> selectors, boolean checkLa
           }
 
           // Now check the label or attribute value
-          if (hasLabel && checkLabels) {
```

Comment:
if `hasLabel`, then `checkLabels` because `hasLabel` can be true only if `checkLabels`